sdk/tables/Azure.Data.Tables/tests/TableServiceLiveTestsBase.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Data.Tables.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="TableServiceClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    public class TableServiceLiveTestsBase : RecordedTestBase<TablesTestEnvironment>
+    {
+
+        public TableServiceLiveTestsBase(bool isAsync, RecordedTestMode recordedTestMode) : base(isAsync, recordedTestMode)
+        {
+            Sanitizer = new TablesRecordedTestSanitizer();
+        }
+
+        public TableServiceLiveTestsBase(bool isAsync) : base(isAsync)
+        {
+            Sanitizer = new TablesRecordedTestSanitizer();
+        }
+
+        protected TableServiceClient service { get; private set; }
+        protected TableClient client { get; private set; }
+        protected string tableName { get; private set; }
+        protected const string PartitionKeyValue = "somPartition";
+
+        /// <summary>
+        /// Creates a <see cref="TableServiceClient" /> with the endpoint and API key provided via environment
+        /// variables and instruments it to make use of the Azure Core Test Framework functionalities.
+        /// </summary>
+        [SetUp]
+        public async Task TablesTestSetup()
+        {
+            service = InstrumentClient(new TableServiceClient(new Uri(TestEnvironment.StorageUri),
+                                                               new TableSharedKeyCredential(
+                                                                   TestEnvironment.AccountName,
+                                                                   TestEnvironment.PrimaryStorageAccountKey
+                                                               ), Recording.InstrumentClientOptions(new TableClientOptions())));
```

Comment:
The closing paren on its own line doesn't match the rest of our style.   I'm surprised we don't have linting rules that complained about this.